eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -17,6 +17,7 @@
 use eZ\Publish\API\Repository\Exceptions\NotFoundException;
 use Exception;
 use eZ\Publish\API\Repository\Values\Content\LocationUpdateStruct;
+use eZ\Publish\Core\Repository\Helper\DomainMapper;
```

Comment:
Actually, integration tests should know nothnbg about core, especially helpers like DomainMappers, so this is a code smell.

Reply:
I've reverted this change (8995f87f80bb5d293285bac4b2fc7c0be44e0168)